components/App.js

```diff
@@ -102,8 +220,10 @@ export const onclickBindings = {
       visible: false,
```

Comment:
We should not reset such a large object; simply clear the values we need to clear. Otherwise this has a dependency on the structure initializer, which is hidden to the programmer.